### PR TITLE
Food api docker

### LINKED ISSUE #10: Food Items Backend
A db with the following fields:

- foodId
- foodName
- dietType
- description
- specialIngredients
- serving
- specialNote
- foodImageId
- locationLat
- locationLong
- status:
- createdAt
- deletedAt
- updatedAt

The backend should allow for manipulation of the status field.

### PR DESCRIPTION
**Pull request by SammyOina**

### CODE REVIEW

**Comment by rodneyosodo** [comment]
This seems good

**Comment by rodneyosodo** [comment]
On the large is the docker image?

**Comment by SammyOina** [comment]
haven't checked will check and try to make it smaller but Gin provides very fast responses and performance is really good

**Comment by SammyOina** [comment]
size is 1.27gb will try sliming or you car try as well

**Comment by rodneyosodo** [comment]
> 
> 
> size is 1.27gb will try sliming or you car try as well

This is a whole operating system😂

**Comment by SammyOina** [comment]
😂 working on it

**Comment by SammyOina** [comment]
maybe big because of database

**Comment by rodneyosodo** [comment]
In your docker file you aren't building a binary?

**Comment by SammyOina** [comment]
the command runs the go file and has to build it, I think its big because of the image having golang image as well as the database

**Comment by SammyOina** [comment]
shrunk it down to 15.2mb

**Comment by rodneyosodo** [comment]
Yes if you compile the application you take the binary and run it in a scratch image instead of golang

**Review by rodneyosodo — APPROVED** [review]
👍🏽

**Comment by SammyOina** [comment]
weird thing is it's smaller that the .exe, 17.7mb, file

**Comment by rodneyosodo** [comment]
> 
> 
> weird thing is it's smaller that the .exe, 17.7mb, file

😂😂

File: Src/Backend/foodApi/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,61 @@
...
-
+# https://blog.golang.org/docker
+# Start from a Debian image with the latest version of Go installed
+# and a workspace (GOPATH) configured at /go.
+# FROM s390x/golang:1.14-alpine
+############################
+# STEP 1 build executable binary
+############################
+# golang alpine 1.13.5
+FROM golang@sha256:0991060a1447cf648bab7f6bb60335d1243930e38420bee8fec3db1267b84cfa as builder
```
Comment (rodneyosodo) [inline Src/Backend/foodApi/Dockerfile:1]: Remove the hashed tag. Replace with a version number or latest

Comment (SammyOina) [reply Src/Backend/foodApi/Dockerfile:1]: using a trusted docker image provides better security could also use golang:alpine
I have removed the comments

**Review by rodneyosodo — APPROVED** [review]
👍

**Comment by rodneyosodo** [comment]
@zejun-zheng @WashingtonKK 
## Your reviews are needed

**Comment by rodneyosodo** [comment]
@SammyOina rebase

**Pull request merged**